news/2018-05-21-project-loved-week-of-may-20th.md

```diff
@@ -0,0 +1,240 @@
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/osu/gid-v.jpg)](https://osu.ppy.sh/community/forums/topics/749054)
+
+[Traktion - Gid V](https://osu.ppy.sh/beatmapsets/56886) by [galvenize](https://osu.ppy.sh/users/381444)
+
+Hopping all the way back to 2012, [galvenize](https://osu.ppy.sh/users/381444)'s Gid V was one of the most popular DnB maps of the time. It's incorporates everything people love about galvenize's style and puts it to a fun, catchy Traktion track. There's very little to say about the map, aside from highlighting its popularity from the time it was submitted.
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/osu/drop-pop-candy.jpg)](https://osu.ppy.sh/community/forums/topics/749055)
+
+[GigaP & Reol - Drop Pop Candy](https://osu.ppy.sh/beatmapsets/194576) by [- M a o n -](https://osu.ppy.sh/users/701533)
+
+If it weren't for the anomaly [Mayday](https://osu.ppy.sh/beatmapsets/756794) this week, [- M a o n -](https://osu.ppy.sh/users/701533)'s Drop Pop Candy would have had the most favorites with 244. For those looking for something easier to play this Loved round, Drop Pop Candy is perfect for you. Its popularity may stem from being one of the first maps following the release of the GigaP/Reol collab song, though its mapping should not be undermined as it matches the pace perfectly.
+
+## <a name="osutaiko" id="osutaiko"></a>osu!taiko
+
+[![](/wiki/shared/news/2018-05-21-project-loved-week-of-may-20th/taiko/still-scared-of-ouija-boards.jpg)](https://osu.ppy.sh/community/forums/topics/749060)
+
+[Goreshit - Still scared of ouija boards](https://osu.ppy.sh/beatmapsets/663341) by [Backfire](https://osu.ppy.sh/users/263110)  
+*written by [Gezoda](https://osu.ppy.sh/users/481582)*
+
+It may not be Halloween yet, but [Backfire](https://osu.ppy.sh/users/263110) and [-Kazu-](https://osu.ppy.sh/users/920861) are still bringing the ouija boards for you to be spooked! Although the song doesn't quite fit the theme, this very awesome beatmap to a great tune really brings out the best of both of these mappers. It’s a good stamina and reading challenge, and it’s not something you want to miss! Just, don’t burn the map if you’re too scared...
```

Comment:
:eyes: 
`’` -> `'`

There are more occurrences throughout, but this was the first one I saw.